DigitalLearningSolutions.Data.Migrations/202111081225_ReduceGroupsGroupDescriptionLength.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace DigitalLearningSolutions.Data.Migrations
+{
+    using FluentMigrator;
+
+    [Migration(202111081225)]
+    public class ReduceGroupsGroupDescriptionLength : Migration
+    {
+        public override void Up()
+        {
+            Execute.Sql("UPDATE Groups SET GroupDescription = LEFT(GroupDescription, 1000)");
+            Alter.Table("Groups").AlterColumn("GroupDescription").AsString(1000).Nullable();

```

Comment:
Does this specify the number of characters or the number of bytes?

Reply:
it'll set it to a nvarchar(1000)

Reply:
Grand. Are you confident you can fit a 1000-character string in there? I'm slightly hazy on this, especially where character encodings are concerned, but if you're confident / have tested then merge away.

Reply:
Yes, I've put a 1000 character string xxxxxxxxx... into mine, and it truncated the longer one down.